runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -94,11 +94,22 @@ default boolean dontHideUntradeables()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "highlightUntradables",
```

Comment:
Fix indentation (https://github.com/runelite/runelite/wiki/Code-Conventions)

Reply:
Gotta love that auto-reformat